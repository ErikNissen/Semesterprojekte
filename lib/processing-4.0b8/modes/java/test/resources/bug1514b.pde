version https://git-lfs.github.com/spec/v1
oid sha256:3a1f9bf01be2077db4d57e276e3a8620aae6e11102141ef0802fe4e6461d16c3
size 218
