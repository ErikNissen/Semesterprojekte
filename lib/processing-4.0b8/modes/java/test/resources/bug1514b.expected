version https://git-lfs.github.com/spec/v1
oid sha256:720df1bb32f398bf2b7e4f07bc0eaf3fad143e6ccec39943fea2e9d0c7be6ef2
size 975
